src/pages/admin/_components/EntityModal.svelte

```diff
@@ -36,16 +35,16 @@ const onCancel = (event: Event) => {
   <Form on:submit={onSubmit}>
     <div class="my-1">
       <p>
-        <TextFieldWithLabel required label="Code" bind:value={formData.code} />
+        <TextField required label="Code" bind:value={formData.code} />
       </p>
       <p>
-        <TextFieldWithLabel required label="Name" bind:value={formData.name} />
+        <TextField required label="Name" bind:value={formData.name} />
       </p>
       <p>
-        <TextFieldWithLabel required label="Income Account" bind:value={formData.income_account} />
+        <TextField required label="Income Account" bind:value={formData.income_account} />
       </p>
       <p>
-        <TextFieldWithLabel label="Parent Entity" bind:value={formData.parent_entity} />
+        <TextField label="Parent Entity" bind:value={formData.parent_entity} />
```

Comment:
```suggestion
        <TextField label="Parent Entity (optional)" bind:value={formData.parent_entity} />
```

As a rule, every field is required unless it's marked otherwise, so we need to mark optional fields as such on the label. Alternatively, we could append `" (optional)"` if `!required` to the label in _ui-components_.